src/app/Events/EventTypes.tsx

```diff
@@ -255,7 +255,7 @@ export const EventTypes: React.FC<EventTypesProps> = (_) => {
   if (errorMessage != '') {
     return (
       <ErrorView
-        title={'Error retrieving event types'}
+        title={'Error retrieving Event Types'}
```

Comment:
same here about `event types`

Reply:
I was matching how the UI looks.
![image](https://github.com/cryostatio/cryostat-web/assets/56743004/66e66afb-e887-45c7-8cba-c41c5ce27aae)

Reply:
I think in this case, that's `Event Types` as title-case because it's a table title. In the error message it doesn't seem there is a particular need to capitalize because it isn't a title, isn't directly referring to the tab as a UI element distinguished by its title, and isn't referring to a resource or specific concept.